node-infrastructure/run-a-collator/collator.md

```diff
@@ -0,0 +1,544 @@
+  ```bash
+  chain-spec-builder create \
+    --relay-chain polkadot \
+    --para-id 1000 \
+    --runtime target/release/wbuild/asset-hub-polkadot-runtime/asset_hub_polkadot_runtime.compact.compressed.wasm \
+    named-preset production > chain-spec.json
+  ```
+
+??? tip "System Parachain Para IDs"
+
+    - Polkadot Hub: 1000
+    - Bridge Hub: 1002
+    - People Chain: 1004
+    - Coretime Chain: 1005
+
+## Create User and Directory Structure
+
+1. Create a dedicated user with the following command:
+  ```bash
+  sudo useradd -r -s /bin/bash polkadot
+  ```
+
+2. Use the following command to copy your chain spec to the directory:
+  ```bash
+  sudo cp chain-spec.json /var/lib/polkadot-collator/
+  ```
+
+3. Set permissions using the following command:
+  ```bash
+  sudo chown -R polkadot:polkadot /var/lib/polkadot-collator
+  ```
+
+## Create Systemd Service File
+
+1. Create a service file to hold the configuration for your collator:
+  ```bash
+  sudo nano /etc/systemd/system/polkadot-collator.service
+  ```
+
+2. Open the new file and add the following configuration code:
+  ```ini title="systemd/system/polkadot-collator.service"
+  [Unit]
+  Description=Polkadot System Parachain Collator
+  After=network.target
+
+  [Service]
+  Type=simple
+  User=polkadot
+  Group=polkadot
+  WorkingDirectory=/var/lib/polkadot-collator
+
+  # Block-Producing Collator Configuration
+  ExecStart=/usr/local/bin/polkadot-omni-node \
+    --collator \
+    --chain=/var/lib/polkadot-collator/chain-spec.json \
+    --base-path=/var/lib/polkadot-collator \
+    --port=30333 \
+    --rpc-port=9944 \
+    --prometheus-port=9615 \
+    --node-key-file=/var/lib/polkadot-collator/node.key \
+    --name="YourCollatorName" \
+    -- \
+    --execution=wasm \
+    --chain=polkadot \
+    --port=30334 \
+    --sync=warp
+
+  Restart=always
+  RestartSec=10
+  LimitNOFILE=65536
+
+  [Install]
+  WantedBy=multi-user.target
+  ```
+
+??? note "Configuration notes"
+
+    - `--collator`: Enables block production mode
+    - `--node-key-file`: Uses the generated node key for stable peer ID
+    - `--name`: Your collator name (visible in telemetry)
+    - Relay chain uses `--sync=warp` for faster initial sync
+
+## Run the Collator
+
+Follow these steps to run your collator node:
+
+1. Reload systemd using the following command:
+  ```bash
+  sudo systemctl daemon-reload
+  ```
+
+2. Next, enable the service to start on boot using the command:
+  ```bash
+  sudo systemctl enable polkadot-collator
+  ```
+3. Now, start the service with the following command:
+  ```bash
+  sudo systemctl start polkadot-collator
+  ```
+
+4. Finally, you can check the status of the service using the command:
+  ```bash
+  sudo systemctl status polkadot-collator
+  ```
+
+To view collator service logs, use the command:
+```bash
+sudo journalctl -u polkadot-collator -f
+```
+
+## Complete Initial Sync
+
+Your collator must sync both the relay chain and parachain before producing blocks.
```

Comment:
I'm confused as to why we are telling them they must do this, but we give no steps/guidance as to how to do it. Is there a guide somewhere else we can at least point them toward?

Reply:
When they run the collator it will do this automatically.